WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -1911,6 +1931,144 @@ static GSourceFuncs _handlerIntervention =
             return _page;
         }
 #endif
+
+        std::vector<std::string> splitString(const std::string &s, char delim)
```

Comment:
This is helper function that can be moved outside of WebKitImplementation. right after CloseDown

Reply:
Yes. I moved the location of this api.